javascript/javascript1/week2/homework/main-3.js

```diff
@@ -0,0 +1,18 @@
+//WEATHER WEAR
+function chooseClothes(tempa) {
+    if (tempa < -5) {
+        return ("Brrr! It's really cold outside. You should wear down jacket, sweatshirt and hat.")
```

Comment:
The parentheses around the string are not needed (though they won't do any harm, either).
Make sure you have installed the "Prettier" extension and set it to format on save, as it will fix these little styling issues:
```
- Open VSCode Settings
- Search 'format'
- Change 'Editor: Default Formatter' to 'Prettier - code formatter'
- Activate 'Editor: Format on save'
```